content/guides/exporters/supported-exporters/Go/ApplicationInsights.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Azure Monitor"
+date: 2018-10-15
```

Comment:
update date. Looks like copy/paste?

Reply:
No, this is actually correct, I did both on the same day. I'll update it now I guess.